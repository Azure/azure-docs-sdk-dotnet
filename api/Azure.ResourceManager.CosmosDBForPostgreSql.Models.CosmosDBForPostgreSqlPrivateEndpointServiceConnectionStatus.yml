### YamlMime:NetType
type: struct
implementsWithMoniker:
- value: <xref href="System.IEquatable`1?alt=System.IEquatable&text=IEquatable" data-throw-if-not-resolved="True"/>&lt;<xref href="Azure.ResourceManager.CosmosDBForPostgreSql.Models.CosmosDBForPostgreSqlPrivateEndpointServiceConnectionStatus?alt=Azure.ResourceManager.CosmosDBForPostgreSql.Models.CosmosDBForPostgreSqlPrivateEndpointServiceConnectionStatus&text=CosmosDBForPostgreSqlPrivateEndpointServiceConnectionStatus" data-throw-if-not-resolved="True"/>&gt;
inheritancesWithMoniker:
- values:
  - <xref href="System.ValueType?alt=System.ValueType&text=ValueType" data-throw-if-not-resolved="True"/>
constructors:
- uid: Azure.ResourceManager.CosmosDBForPostgreSql.Models.CosmosDBForPostgreSqlPrivateEndpointServiceConnectionStatus.#ctor(System.String)
  monikers:
  - azure-dotnet-preview
operators:
- uid: Azure.ResourceManager.CosmosDBForPostgreSql.Models.CosmosDBForPostgreSqlPrivateEndpointServiceConnectionStatus.op_Equality(Azure.ResourceManager.CosmosDBForPostgreSql.Models.CosmosDBForPostgreSqlPrivateEndpointServiceConnectionStatus,Azure.ResourceManager.CosmosDBForPostgreSql.Models.CosmosDBForPostgreSqlPrivateEndpointServiceConnectionStatus)
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.CosmosDBForPostgreSql.Models.CosmosDBForPostgreSqlPrivateEndpointServiceConnectionStatus.op_Implicit(System.String)~Azure.ResourceManager.CosmosDBForPostgreSql.Models.CosmosDBForPostgreSqlPrivateEndpointServiceConnectionStatus
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.CosmosDBForPostgreSql.Models.CosmosDBForPostgreSqlPrivateEndpointServiceConnectionStatus.op_Inequality(Azure.ResourceManager.CosmosDBForPostgreSql.Models.CosmosDBForPostgreSqlPrivateEndpointServiceConnectionStatus,Azure.ResourceManager.CosmosDBForPostgreSql.Models.CosmosDBForPostgreSqlPrivateEndpointServiceConnectionStatus)
  monikers:
  - azure-dotnet-preview
methods:
- uid: Azure.ResourceManager.CosmosDBForPostgreSql.Models.CosmosDBForPostgreSqlPrivateEndpointServiceConnectionStatus.Equals(Azure.ResourceManager.CosmosDBForPostgreSql.Models.CosmosDBForPostgreSqlPrivateEndpointServiceConnectionStatus)
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.CosmosDBForPostgreSql.Models.CosmosDBForPostgreSqlPrivateEndpointServiceConnectionStatus.ToString
  monikers:
  - azure-dotnet-preview
properties:
- uid: Azure.ResourceManager.CosmosDBForPostgreSql.Models.CosmosDBForPostgreSqlPrivateEndpointServiceConnectionStatus.Approved
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.CosmosDBForPostgreSql.Models.CosmosDBForPostgreSqlPrivateEndpointServiceConnectionStatus.Pending
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.CosmosDBForPostgreSql.Models.CosmosDBForPostgreSqlPrivateEndpointServiceConnectionStatus.Rejected
  monikers:
  - azure-dotnet-preview
uid: Azure.ResourceManager.CosmosDBForPostgreSql.Models.CosmosDBForPostgreSqlPrivateEndpointServiceConnectionStatus
commentId: T:Azure.ResourceManager.CosmosDBForPostgreSql.Models.CosmosDBForPostgreSqlPrivateEndpointServiceConnectionStatus
namespace: Azure.ResourceManager.CosmosDBForPostgreSql.Models
name: CosmosDBForPostgreSqlPrivateEndpointServiceConnectionStatus
fullName: Azure.ResourceManager.CosmosDBForPostgreSql.Models.CosmosDBForPostgreSqlPrivateEndpointServiceConnectionStatus
nameWithType: Azure.ResourceManager.CosmosDBForPostgreSql.Models.CosmosDBForPostgreSqlPrivateEndpointServiceConnectionStatus
assembliesWithMoniker:
- value: Azure.ResourceManager.CosmosDBForPostgreSql.dll
packagesWithMoniker:
- value: Azure.ResourceManager.CosmosDBForPostgreSql v1.0.0-beta.1
syntaxWithMoniker:
- lang: csharp
  values:
  - value: 'public readonly struct CosmosDBForPostgreSqlPrivateEndpointServiceConnectionStatus : IEquatable<Azure.ResourceManager.CosmosDBForPostgreSql.Models.CosmosDBForPostgreSqlPrivateEndpointServiceConnectionStatus>'
- lang: fsharp
  values:
  - value: type CosmosDBForPostgreSqlPrivateEndpointServiceConnectionStatus = struct
- lang: vb
  values:
  - value: >-
      Public Structure CosmosDBForPostgreSqlPrivateEndpointServiceConnectionStatus

      Implements IEquatable(Of CosmosDBForPostgreSqlPrivateEndpointServiceConnectionStatus)
devLangs:
- csharp
- vb
- fsharp
monikers:
- azure-dotnet-preview
summary: The private endpoint connection status.
metadata:
  api_name:
  - Azure.ResourceManager.CosmosDBForPostgreSql.Models.CosmosDBForPostgreSqlPrivateEndpointServiceConnectionStatus
  api_location:
  - Azure.ResourceManager.CosmosDBForPostgreSql.dll
  topic_type:
  - apiref
  api_type:
  - Assembly
  f1_keywords:
  - Azure.ResourceManager.CosmosDBForPostgreSql.Models.CosmosDBForPostgreSqlPrivateEndpointServiceConnectionStatus
  - Azure::ResourceManager::CosmosDBForPostgreSql::Models::CosmosDBForPostgreSqlPrivateEndpointServiceConnectionStatus
  - CosmosDBForPostgreSqlPrivateEndpointServiceConnectionStatus
  helpviewer_keywords:
  - Azure.ResourceManager.CosmosDBForPostgreSql.Models.CosmosDBForPostgreSqlPrivateEndpointServiceConnectionStatus struct [.NET]
  monikers:
  - azure-dotnet-preview
