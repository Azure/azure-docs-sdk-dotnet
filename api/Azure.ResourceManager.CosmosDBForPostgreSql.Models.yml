### YamlMime:NetNamespace
classes:
- uid: Azure.ResourceManager.CosmosDBForPostgreSql.Models.ArmCosmosDBForPostgreSqlModelFactory
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.CosmosDBForPostgreSql.Models.CosmosDBForPostgreSqlClusterNameAvailabilityContent
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.CosmosDBForPostgreSql.Models.CosmosDBForPostgreSqlClusterNameAvailabilityResult
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.CosmosDBForPostgreSql.Models.CosmosDBForPostgreSqlClusterPatch
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.CosmosDBForPostgreSql.Models.CosmosDBForPostgreSqlMaintenanceWindow
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.CosmosDBForPostgreSql.Models.CosmosDBForPostgreSqlPrivateLinkServiceConnectionState
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.CosmosDBForPostgreSql.Models.CosmosDBForPostgreSqlServerRoleGroupConfiguration
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.CosmosDBForPostgreSql.Models.CosmosDBForPostgreSqlSimplePrivateEndpointConnection
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.CosmosDBForPostgreSql.Models.ServerNameItem
  monikers:
  - azure-dotnet-preview
structs:
- uid: Azure.ResourceManager.CosmosDBForPostgreSql.Models.CosmosDBForPostgreSqlConfigurationDataType
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.CosmosDBForPostgreSql.Models.CosmosDBForPostgreSqlNameAvailabilityResourceType
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.CosmosDBForPostgreSql.Models.CosmosDBForPostgreSqlPrivateEndpointConnectionProvisioningState
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.CosmosDBForPostgreSql.Models.CosmosDBForPostgreSqlPrivateEndpointServiceConnectionStatus
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.CosmosDBForPostgreSql.Models.CosmosDBForPostgreSqlServerRole
  monikers:
  - azure-dotnet-preview
- uid: Azure.ResourceManager.CosmosDBForPostgreSql.Models.ProvisioningState
  monikers:
  - azure-dotnet-preview
uid: Azure.ResourceManager.CosmosDBForPostgreSql.Models
commentId: N:Azure.ResourceManager.CosmosDBForPostgreSql.Models
name: Azure.ResourceManager.CosmosDBForPostgreSql.Models
fullName: Azure.ResourceManager.CosmosDBForPostgreSql.Models
devLangs:
- csharp
monikers:
- azure-dotnet-preview
metadata:
  f1_keywords:
  - Azure.ResourceManager.CosmosDBForPostgreSql.Models
  - Azure::ResourceManager::CosmosDBForPostgreSql::Models
  - Models
  - CosmosDBForPostgreSql.Models
  - CosmosDBForPostgreSql::Models
  helpviewer_keywords:
  - Azure.ResourceManager.CosmosDBForPostgreSql.Models namespace [.NET]
  monikers:
  - azure-dotnet-preview
