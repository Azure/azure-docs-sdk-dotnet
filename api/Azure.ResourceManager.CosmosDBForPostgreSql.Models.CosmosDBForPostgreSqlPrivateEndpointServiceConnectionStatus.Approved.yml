### YamlMime:NetMember
type: property
members:
- returnsWithMoniker:
    type:
    - value: <xref href="Azure.ResourceManager.CosmosDBForPostgreSql.Models.CosmosDBForPostgreSqlPrivateEndpointServiceConnectionStatus?alt=Azure.ResourceManager.CosmosDBForPostgreSql.Models.CosmosDBForPostgreSqlPrivateEndpointServiceConnectionStatus&text=CosmosDBForPostgreSqlPrivateEndpointServiceConnectionStatus" data-throw-if-not-resolved="True"/>
  type: property
  uid: Azure.ResourceManager.CosmosDBForPostgreSql.Models.CosmosDBForPostgreSqlPrivateEndpointServiceConnectionStatus.Approved
  commentId: P:Azure.ResourceManager.CosmosDBForPostgreSql.Models.CosmosDBForPostgreSqlPrivateEndpointServiceConnectionStatus.Approved
  name: Approved
  fullName: Azure.ResourceManager.CosmosDBForPostgreSql.Models.CosmosDBForPostgreSqlPrivateEndpointServiceConnectionStatus.Approved
  nameWithType: CosmosDBForPostgreSqlPrivateEndpointServiceConnectionStatus.Approved
  syntaxWithMoniker:
  - lang: csharp
    values:
    - value: public static Azure.ResourceManager.CosmosDBForPostgreSql.Models.CosmosDBForPostgreSqlPrivateEndpointServiceConnectionStatus Approved { get; }
  - lang: fsharp
    values:
    - value: 'static member Approved : Azure.ResourceManager.CosmosDBForPostgreSql.Models.CosmosDBForPostgreSqlPrivateEndpointServiceConnectionStatus'
  - lang: vb
    values:
    - value: Public Shared ReadOnly Property Approved As CosmosDBForPostgreSqlPrivateEndpointServiceConnectionStatus
  monikers:
  - azure-dotnet-preview
  summary: Approved.
  metadata: {}
uid: Azure.ResourceManager.CosmosDBForPostgreSql.Models.CosmosDBForPostgreSqlPrivateEndpointServiceConnectionStatus.Approved*
commentId: Overload:Azure.ResourceManager.CosmosDBForPostgreSql.Models.CosmosDBForPostgreSqlPrivateEndpointServiceConnectionStatus.Approved
namespace: Azure.ResourceManager.CosmosDBForPostgreSql.Models
name: Approved
fullName: Azure.ResourceManager.CosmosDBForPostgreSql.Models.CosmosDBForPostgreSqlPrivateEndpointServiceConnectionStatus.Approved
nameWithType: CosmosDBForPostgreSqlPrivateEndpointServiceConnectionStatus.Approved
assembliesWithMoniker:
- value: Azure.ResourceManager.CosmosDBForPostgreSql.dll
packagesWithMoniker:
- value: Azure.ResourceManager.CosmosDBForPostgreSql v1.0.0-beta.1
devLangs:
- csharp
- vb
- fsharp
monikers:
- azure-dotnet-preview
metadata:
  api_name:
  - Azure.ResourceManager.CosmosDBForPostgreSql.Models.CosmosDBForPostgreSqlPrivateEndpointServiceConnectionStatus.Approved
  - Azure.ResourceManager.CosmosDBForPostgreSql.Models.CosmosDBForPostgreSqlPrivateEndpointServiceConnectionStatus.get_Approved
  api_location:
  - Azure.ResourceManager.CosmosDBForPostgreSql.dll
  topic_type:
  - apiref
  api_type:
  - Assembly
  f1_keywords:
  - Azure.ResourceManager.CosmosDBForPostgreSql.Models.CosmosDBForPostgreSqlPrivateEndpointServiceConnectionStatus.Approved
  - Azure::ResourceManager::CosmosDBForPostgreSql::Models::CosmosDBForPostgreSqlPrivateEndpointServiceConnectionStatus::Approved
  - Approved
  - CosmosDBForPostgreSqlPrivateEndpointServiceConnectionStatus.Approved
  - CosmosDBForPostgreSqlPrivateEndpointServiceConnectionStatus::Approved
  helpviewer_keywords:
  - CosmosDBForPostgreSqlPrivateEndpointServiceConnectionStatus.Approved property [.NET]
  - Approved property [.NET], struct CosmosDBForPostgreSqlPrivateEndpointServiceConnectionStatus
  monikers:
  - azure-dotnet-preview
